### ISSUE TITLE
Incomplete AWS megatests results 

### ISSUE THREAD

**Issue by drejom** [nf-core/smrnaseq#219]
### Description of the bug

The AWS megatest results typically posted at https://nf-co.re/smrnaseq/results are incomplete. 

Expecting folders for:
```
edger
fastp
fastqc
genome
index
mirdeep
mirdeep2
mirtop
mirtrace
multiqc
pipeline_info
samtools
unmapped
```

Instead there's only: 
```
fastqc
genome
index
pipeline_info
```

### System information

nf-core/smrnaseq v2.1.0

**Comment by apeltzer** [comment]
Will check this before making the next patch release - something odd, should not be the case - thanks for reporting!

**Comment by apeltzer** [comment]
Fixed in 2.2.0 and now also in 2.2.1
